packages/universal-crossplane/overlay/questions.yaml

```diff
@@ -0,0 +1,6 @@
+questions: 
+- variable: upbound.controlPlane.token
+  required: false
+  type: password
+  label: Control plane token to connect Upbound Cloud
+  group: "Upbound"
```

Comment:
Please change the group name to "Settings" or something other than the chart's name. This is displayed in the UI as a tab on the side of your chart's installation page for users to configure your chart.

Reply:
@PennyScissors this was actually changed as "Upbound Cloud" in a later commit of this PR. This section contains settings regarding connecting uxp to Upbound Cloud, hence "Upbound Cloud" as a group name sounds good.

Here is how it is rendered on my rancher instance:
<img width="1417" alt="Screen Shot 2021-06-14 at 11 27 18" src="https://user-images.githubusercontent.com/9900707/121862506-87b03480-cd03-11eb-868b-bd6ccb1c4d2e.png">